src/components/header/index.js

```diff
@@ -54,14 +54,14 @@ class Header extends React.Component {
   render() {
     return <div style={{height: '72px'}}>
       <nav className="uk-navbar-container uk-margin uk-dark uk-navbar-transparent" uk-navbar="mode: click">
-        <div className="uk-navbar-center">
+        <div className="uk-navbar-right right">
```

Comment:
Try adding least custom CSS classes.
For eg: instead of `right` use `uk-margin-xlarge-right`

Reply:
Done the changes @aashutoshrathi